### PR TITLE
LPD-20497 Adjust cards positioning and create recipient component

### PR DESCRIPTION
**Pull request by larissacribeiro**
[LPD-20497](https://liferay.atlassian.net/browse/LPD-20497), [LPD-20548](https://liferay.atlassian.net/browse/LPD-20548) and [LPD-20547](https://liferay.atlassian.net/browse/LPD-20547)

### CODE REVIEW

**Comment by igor-franca** [comment]
Hey @larissacribeiro, Jenkins Console:

> The commit message is missing a reference to a required JIRA project

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @larissacribeiro, Jenkins Console:

> The commit message is missing a reference to a required JIRA project

**Comment by larissacribeiro** [comment]
Hey @igor-franca, my bad, just fixed it. thanks!

**Comment by larissacribeiro** [comment]
ci:test:sf

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/RecipientFields.tsx
Comment (igor-franca) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/RecipientFields.tsx:1]: Hey Larissa, I thought I'd change things up a little here.
I thought we could add a more generic `RecipientPanel` component, like:

```
    return (
        <ClayPanel
            displayTitle={PanelTitle}
            displayType="unstyled"
        >
            <ClayPanel.Body>
                {children}
            </ClayPanel.Body>
         </ClayPanel>
    )
```

Which must receive the child components PrimaryRecipients and SecodRecipients, like:

```
<RecipientPanel>
	<PrimaryRecipient
		errors={errors}
		recipientOptions={RECIPIENT_OPTIONS}
		selectedLocale={selectedLocale}
		setValues={setValues}
		values={values}
	/>
</RecipientPanel>
```

Comment (igor-franca) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/RecipientFields.tsx:1]: I would like to hear what @danvb112 has to say about this, before we move on to Larissa's changes.

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey larissa, great work here. Just a few points

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
I've validated with Lawrence from the product team the change of the link text to be **Learn More** instead of **View Documentation**, as it is in most of the learn-resources, since I learned from Brian in a previous attempt of mine in using _LiferayLearn_, to avoid words like "Click" and "Documentation".

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -15545,6 +15547,7 @@ search-file-entries=Search File Entries
 search-folder=Search Folder
 search-folders=Search Folders
 search-for=Search for
+search-for-a-role=Search for a role
```
Comment (igor-franca) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:15550]: Hey, Dan and Larissa.

I know that some of these languages ​​will be used as placeholders, but can we check if we should use title case or sentence case for these cases?

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -19842,6 +19846,7 @@ type-a-comma-or-press-enter-to-input-email-addresses=Type a comma or press enter
 type-a-comma-or-press-enter-to-input-roles=Type a comma or press enter to input roles.
 type-a-single-email-address=Type a single email address:
 type-can-only-be-email=Type can only be email.
+type-email-address=Type email adress
```
Comment (igor-franca) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:19849]: Do the same here :)

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx [hunk 3]
```diff
@@ -5,165 +5,110 @@
...
 import {FormError, Input} from '@liferay/object-js-components-web';
 import {InputLocalized} from 'frontend-js-components-web';
 import React from 'react';
 
 import {NotificationTemplateError} from '../EditNotificationTemplate';
+import {PrimaryRecipient} from './PrimaryRecipients';
+import {SecondaryRecipient} from './SecondaryRecipients';
+import {Sender} from './Sender';
 
+import './EmailNotificationSettings.scss';
```
Comment (igor-franca) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx:22]: Hey @larissacribeiro, could you add a blank line here?

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx [hunk 4]
```diff
@@ -91,7 +185,7 @@ export function PrimaryRecipient({
 				<ClayTooltipProvider>
 					<span
 						title={Liferay.Language.get(
-							'each-to-recipient-will-receive-separate-emails'
+							'each-to-primaryRecipient-will-receive-separate-emails'
```
Comment (igor-franca) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx]: Hey @larissacribeiro, could you confirm if this change was intencional?

File: learn-resources/data/notification-web.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+	"general": {
+		"en_US": {
+			"message": "Learn more.",
```
Comment (igor-franca) [inline learn-resources/data/notification-web.json:4]: Hey @larissacribeiro, could you check if this is the right message?

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @larissacribeiro, nice work here! Just some details. 

I didn't have the time necessary to carry out the tests I wanted, I would like to confirm if @danvb112  can do a more complete test of the story before sending it to product. Thaks!

**Comment by larissacribeiro** [comment]
ci:test:sf

File: learn-resources/data/notification-web.json (hunk 5 above)
Comment (larissacribeiro) [reply learn-resources/data/notification-web.json:4]: yes, @igor-franca, I've left a comment above yours explaining this change!

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx (hunk 4 above)
Comment (larissacribeiro) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx]: it was not, thanks!

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx (hunk 3 above)
Comment (larissacribeiro) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx:26]: done, thanks!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (larissacribeiro) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:15550]: we decided to treat them as sentences, thanks Igor!

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
Relevant failures are not related.

Objects possible unique failures, comparing with the [last analyzed build](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1712365921757&testrayRunIdA=2558796382&testrayRunIdB=2558948641):
**1. object-web/modelBuilder.spec.ts > can delete object relationship from different folders
2. LocalFile.ObjectActions#CanCreateActionAddObjectEntryWithOnPaymentStatusUpdate
3. LocalFile.ObjectPermission#CannotViewSingleObjectWithoutViewOwnerPermission
4. LocalFile.ObjectActions#CanCreateActionUsingGroovyScripWithOnOrderStatusUpdate**

**Review by danvb112 — APPROVED** [review]
LGTM!

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:foward:force

**Comment by larissacribeiro** [comment]
ci:forward:force

**Comment by larissacribeiro** [comment]
ci:forward:force

**Comment by larissacribeiro** [comment]
ci:forward:force

**Comment by larissacribeiro** [comment]
I will address the conflicts before trying to forward again.

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:forward:force

**Pull request closed without merging**